compiler/src/org.graalvm.compiler.asm.amd64/src/org/graalvm/compiler/asm/amd64/AMD64Assembler.java

```diff
@@ -208,11 +203,44 @@ public String toString() {
     }
 
     private static class VexOpcode {
+        // private static final int VEX_OPCODE_NONE = 0x0;
         private static final int VEX_OPCODE_0F = 0x1;
         private static final int VEX_OPCODE_0F_38 = 0x2;
         private static final int VEX_OPCODE_0F_3A = 0x3;
     }
 
+    public static class AvxVectorLen {
+        public static final int AVX_128bit = 0x0;
+        public static final int AVX_256bit = 0x1;
+        public static final int AVX_512bit = 0x2;
+        public static final int AVX_NoVec = 0x4;
+    }
+
+    private static class EvexTupleType {
+        public static final int EVEX_FV = 0;
+        // public static final int EVEX_HV = 4;
```

Comment:
These bits of commented code need to be removed or uncommented when this PR is ready to merge.

Reply:
Style check don't allow unused constants, but it's better to keep whole constant set for readability/future development.

Reply:
You can solve this by making `EvexTupleType` non-private (package-protected should be sufficient to convince Checkstyle that it might be used).